_posts/2024-08-23-GSoC-CSL.md

```diff
@@ -0,0 +1,65 @@
+---
+title: JabRef's new CSL Integration for LibreOffice
+tags: [gsoc, csl]
+author: subhramit
+---
+
+Hi, I am Subhramit. I am a Google Summer of Code (GSoC) student at JabRef this year, and I'll be walking you through JabRef's new CSL Integration for LibreOffice.
+
+## Background
+
+As a student, I first came across pre-defined citation formats during my freshman year of college, when our professor instructed that the references in our project report must follow the "APA style".  
+
+I remember manually generating and copying citations in that style from some citation generation website, and oh was it tedious. Whether you are a student like me or a seasoned researcher in academia, you must have grappled with various citation styles throughout your academic journey.
+
+[Citation Style Language](https://citationstyles.org/) (CSL) is a popular open-source specification language that standardizes the formatting of citations and bibliographies. It supports thousands of standard citation styles used in academia, including *American Psychological Association* (APA), *Modern Language Association* (MLA), *Chicago Manual of Style* (CMS), *Institute of Electrical and Electronics Engineers* (IEEE), *Springer - Lecture Notes in Computer Science* and *Vancouver* to name a few. If you've ever cited a source in an academic paper, chances are you've used a CSL-based style.
+
+For several years, JabRef users have been requesting support for CSL in JabRef's LibreOffice integration. Tackling this problem statement this summer as a part of my project, I am excited to announce that this highly anticipated feature is now available!
+
+## What's New?
+
+With this integration, JabRef users can now:
+
+1. **Choose a CSL Style**: Select from a vast collection of CSL styles to match the requirements of your academic work.
+2. **Cite with ease**: Insert citations directly into your LibreOffice document.
+3. **Auto-generate Bibliographies**: Watch as your bibliography automatically populates based on your citations, with just a single click.
+4. **Update Bibliographies**: Easily refresh your bibliography as you add new citations to your document.
+
+## Getting Started
+
+To start using this new feature:
+
+1. Download the [development version of JabRef](https://builds.jabref.org/main/)
+2. Connect to a running LibreOffice document instance from the [Libre/OpenOffice Panel](https://docs.jabref.org/cite/openofficeintegration)
```

Comment:
2. Connect to a running LibreOffice document instance by either clicking the "Connect" or "Manual Connect" button in the [Libre/OpenOffice Panel](https://docs.jabref.org/cite/openofficeintegration) in the side pane.